gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java

```diff
@@ -60,17 +60,17 @@ private String readJsonFile(File file) {
       return sb.toString();
     } catch (IOException e) {
```

Comment:
If you're throwing the exception callers will need to catch it, and they will try to log it as well; in that case no need for the try/catch and `Log`,

Reply:
In fact I see it's logged in the calling function, so ok to remove the try/catch block.

Reply:
Thanks!